researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -430,24 +423,18 @@ public Completable forgotPassword(@NonNull String email) {
     @Override
     @Nullable
     public User getUser(@Nullable Context context) {
-        logger.debug("Called getUser");
-        if (userLocalStorage == null) {
-            return null;
-        }
-        return userLocalStorage.loadUser();
+        return researchStackDAO.getUser();
     }
 
     @Override
     @Nullable
     public void setUser(Context context, User user) {
-        logger.debug("Called getUser");
-        userLocalStorage.saveUser(user);
+        researchStackDAO.setUser(user);
     }
 
     @Override
     public String getUserEmail(Context context) {
-        User user = userLocalStorage.loadUser();
-        return user == null ? null : user.getEmail();
+        return authenticationManager.getEmail();
```

Comment:
Seems odd that email comes from the auth manager, shouldn't it be stored in the user object?